packages/paste-core/components/pagination/src/PaginationLabel.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import {Text, safelySpreadTextProps} from '@twilio-paste/text';
+import type {PaginationLabelProps} from './types';
+import {PaginationLabelPropTypes} from './proptypes';
+
+const PaginationLabel = React.forwardRef<HTMLDivElement, PaginationLabelProps>(({children, ...props}, ref) => {
+  return (
+    <Text
+      {...safelySpreadTextProps(props)}
+      ref={ref}
+      as="div"
+      color="colorTextWeak"
+      paddingBottom="space20"
+      paddingTop="space20"
+      textAlign="center"
+    >
+      {children}
+    </Text>
+  );
+});
+
+PaginationLabel.displayName = 'PaginationLabel';
+
+if (process.env.NODE_ENV === 'development') {
```

Comment:
We totally don't need these checks. We don't strip them out on compile and PropTypes already doesn't run them in production mode.